doc/user/layouts/shortcodes/experimental.html

```diff
@@ -0,0 +1,10 @@
+<div class="experimental">
+  <strong>EXPERIMENTAL!</strong> This feature is under construction and requires
+  <a href="{{"/cli/#experimental-mode"|relURL}}">experimental mode</a>.
+
+  {{if in (apply $.Site.Params.versions "index" "." "name") (.Get 0)}}
+  <em>Available since {{.Get 0}}.</em>
+  {{else}}
+  <em>Available only in unstable developer builds.</em>
```

Comment:
Let's just call these "unstable builds" or call the header at `versions/#unstable-builds` "Unstable Developer Builds" just to ensure that search unambiguously refers to the same thing.

Reply:
You bet, done!